dev-support/create-release/mac-sshd-gpg-agent/Dockerfile

```diff
@@ -0,0 +1,100 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Image for use on Mac boxes to get a gpg agent socket available
```

Comment:
This is unique to mac as the host os? What about a linux host?

Reply:
You could use this on other host OSes but it is only needed on the Mac because Docker Desktop for mac can't bind mount generic sockets.

there's notes in the `do-docker-build.sh` about use of this container on OS X vs what happens when we run on other OSes.